File: Java/Object Validator/lib/src/main/java/com/quicksilver/objectvalidator/runtime/True.java
package com.quicksilver.objectvalidator.runtime;

import java.util.HashSet;

public class True extends Condition {
    public True(boolean reversed, String fieldExpression) {
        super(reversed, fieldExpression);
    }

    @Override
    protected boolean isFulfilledBy(Object value, String fullFieldExpression, HashSet<String> passedFields, HashSet<String> failedFields) {
        return value != null && (boolean)value;
    }
}

File: C Sharp/Object Validator/Runtime/True.cs
namespace Quicksilver.ObjectValidator.Runtime;
class True(bool reversed, string? fieldExpression) : Condition(reversed, fieldExpression)
{
    protected override bool IsFulfilledBy(object? value, string? fullFieldExpression, HashSet<string> passedFields, HashSet<string> failedFields)
    {
        return value is true;
    }
}